### DISCUSSION TITLE
Gamerule Execution from Console

### DISCUSSION THREAD

**Discussion by HackuJacku** [Zrips/CMI#6821] [Enhancement]
Currently, `/gamerule` is unable to be executed from the console due to the usage of the GUI. Upon executing the command, the error "You can only use this in game!" is provided.

I'd like to propose that the `/gamerule` mechanic in GUI is enhanced so that you are able to execute the command from console. As the dimension is specified through the CMI syntax, the command would execute and modify the dimension's gamerule.

**Comment by CloudeLecaw** [comment]
U know u have a minecraft:gamerule command that does this all? The /cmi gamerule is just a visual version ingame of that in form of a gui.

**Reply by HackuJacku** [reply]
> U know u have a minecraft:gamerule command that does this all? The /cmi gamerule is just a visual version ingame of that in form of a gui.

I am completely aware but there are two issues with this method:
1) You can't specify a dimension if using multiple worlds - I want to change the gamerules via console for a specific dim.
2) Even when changing the gamerules via the Minecraft version, the CMI GUI does not update to reflect these new gamerules.
